tests/apptest.go

```diff
@@ -161,6 +162,27 @@ func (t *AppTest) TestH_Delete_SUCCESS() {
 	testSuccess(t, true, "")
 }
 
+func (t *AppTest) TestConfirmMail(){
```

Comment:
The tests run in alphabetic order, maybe we can stick to the existing naming convention, therefore it would be TestI_Confirmation_Mail, seeing as how it's not related to an actual user.